### ISSUE TITLE
Migrate daily service tests to GitHub actions

### ISSUE THREAD

**Issue by chris48s** [badges/shields#8821]
:clipboard: **Description**

Having migrated all the CI on this repo from Circle CI to GHA, now its time to do the same on https://github.com/badges/daily-tests

I think if I was implementing this from scratch today, I would just set up a scheduled job to run on this repo, like this one:

https://github.com/badges/shields/blob/e8c42e9ec1b2d93790133a7843b04f4a9869cec7/.github/workflows/draft-release.yml#L2-L5

@calebcartwright - Can you remember anything about the rationale for running the daily tests from another repo?

I wonder if we should bring them back into the fold? 🤔

**Comment by PyvesB** [comment]
Is it me, or the daily tests are no longer working at all at this point?

**Comment by chris48s** [comment]
Yeah the daily runs on https://github.com/badges/daily-tests no longer work.

I can't remember exactly when they broke, but it was quite a while back. I decided that instead of putting the time into fixing up that old Circle CI config, I'd rather put the time into moving the daily runs onto scheduled GH actions in this repo. They'll be way easier to keep in sync that way, rather than having to coordinate changes across 2 repos.

But then I ended up doing neither of those things.
